test/controllers/address.controller.test.js

```diff
@@ -154,14 +155,18 @@ describe('Address controller', () => {
 
       describe('when the request succeeds', () => {
         beforeEach(() => {
-          Sinon.stub(SubmitSelectAddressService, 'go').returns({})
+          Sinon.stub(SubmitSelectAddressService, 'go').returns({
+            redirect: '/system/notices/setup/fecd5f15-bacf-4b3d-bdcd-ef279a97b061/check',
+            succeeded: true
+          })
+          Sinon.stub(AddAdditionalRecipientService, 'go').returns({})
```

Comment:
The service doesn't return anything so we should reflect this in the test.

```suggestion
          Sinon.stub(AddAdditionalRecipientService, 'go')
```